src/main/java/seedu/address/model/item/Quantity.java

```diff
@@ -42,6 +42,16 @@ public Quantity add(Quantity quantity) {
         return new Quantity(Integer.toString(value));
     }
 
+    /**
+     * subtracts a Quantity's value from another quantity's value
+     * @param quantity another quantity
+     * @return Difference in Quantity
```

Comment:
If I rmb correctly, the following word after @return it should be the type, then description for subsequent words

Reply:
Ok fixed in latest commit